docs/java/tips-and-tricks/logging.md

```diff
@@ -0,0 +1,76 @@
+---
+id: change-log-level
+title: Change log level
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Increase log level
+description: Change log level for your Cloud SDK application for efficient debuggins
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+image:
+---
+
+## When debugging locally ##
+To quickly change `log level` while debugging your Cloud SDK application locally add this argument to you Java compiler:
```

Comment:
Isn't it more the mvn run command instead of the compiler?
```suggestion
To quickly change `log levels` while debugging your Cloud SDK application locally add this argument to your `mvn` run command:
```

Reply:
Also, I'm not sure whether this is actually true, as we (if I remember correctly) only use the slf4j simpleLogger for testing, not for the actual runtime.